app/tools/android/customize.py

```diff
@@ -331,8 +331,7 @@ def CustomizeJava(app_info, app_url, app_local_path, keep_screen_on):
   dest_activity = os.path.join(app_pkg_dir, name + 'Activity.java')
   ReplaceString(dest_activity, 'org.xwalk.app.template', package)
   ReplaceString(dest_activity, 'AppTemplate', name)
-  manifest_file = os.path.join(app_dir, 'assets', 'www', 'manifest.json')
-  if os.path.isfile(manifest_file):
+  if manifest:
```

Comment:
If the file is not called `manifest.json`, isn't the `ReplaceString()` call below going to fail (ie. it'll generate a load call for a file that is not present)?